torch/_export/__init__.py

```diff
@@ -92,14 +84,13 @@ class ExportDynamoConfig:
 DECOMP_TABLE = core_aten_decompositions()
 
 
-def _export(
+def export(
     f: Callable,
-    args: Tuple[Value],
+    args: Tuple[Any],
     constraints: Optional[List[Constraint]] = None,
-) -> torch.fx.GraphModule:
+) -> graph_module.ExportedProgram:
     """
-    Private API to export a single entry point or a free function. It is meant to be used
-    inside top level torch.export.
+    Export a single entry point or a free function.
```

Comment:
Since this is the public API now, we should leave a docstring. Maybe you can repurpose the one that was used in the original export call.